lib/helpers/environment.php

```diff
@@ -0,0 +1,13 @@
+<?php
+/**
+ * Helper functions pretaining to the environment domain
+*/
+
```

Comment:
Let's add a `require_once` for the class here just for extra safety.

Reply:
Thank you. I have made this change in the latest commit.